### PR TITLE
First stab at old-mode support

### PR DESCRIPTION
**Pull request by mbg**
This PR adds _very_ rudimentary support for old-mode SSO, just about enough for an application that uses it (but doesn't use the information from the sentry) to work. (Please don't use this.)

### CODE REVIEW

File: app/views/slogin.scala.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+                @for(grp <- members.groupBy(m => m.department.name).toSeq.sortBy((a) => a._1)) {
+                    <optgroup label="@grp._1">
+                    @for(member <- grp._2) {
+                        <option value="@member.universityId">@member.givenName @member.familyName, @member.universityId, @member.warwickTargetGroup</option>
+                    }
+                    </optgroup>
+                }
+            </select>
+        </div>
+        <button type="submit" class="btn btn-primary">POST to the ACS</button>
```
Comment (lol768) [inline app/views/slogin.scala.html]:
```suggestion
        <button type="submit" class="btn btn-primary">Set cookie and redirect</button>
```

File: app/views/slogin.scala.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
+@import domain.Member
+@(providerId: String, target: String, members: Seq[Member])(implicit context: RequestContext)
+
+@main("Handle Service") {
+    <p><samp>@providerId</samp> wants to perform authentication!</p>
```
Comment (lol768) [inline app/views/slogin.scala.html]:
```suggestion
    <p><samp>@providerId</samp> wants to perform old mode authentication!</p>
```

File: app/controllers/IndexController.scala [hunk 3]
```diff
@@ -112,4 +121,7 @@ class IndexController extends BaseController {
     Ok(scala.xml.XML.loadString(new String(canonicalizer.canonicalizeSubtree(soapEnvelope))))
   }
 
+  def respondToSentry() = Action { implicit request =>
+    Ok("returnType=1\nid=" + request.body.asFormUrlEncoded.get("token"))
```
Comment (lol768) [inline app/controllers/IndexController.scala]: Can we use `fakeMemberService.getResponseFor(member)` to get the attributes based on the cookie value? They should be simple to serialise

**Review by lol768 — CHANGES_REQUESTED** [review]

File: app/controllers/IndexController.scala (hunk 3 above)
Comment (mbg) [reply app/controllers/IndexController.scala]: I can look into it 👍

**Comment by mbg** [comment]
The attributes in `AttributeConverter` don't seem to quite line up with all the ones listed at https://warwick.ac.uk/services/its/servicessupport/web/sign-on/development/reference/ssov2/interfaces/getuserbytoken 

Not sure if that's worth pursuing though?

**Comment by lol768** [comment]
As discussed, some of the new mode attributes are renamed when returning back an old mode response. We can fix this later though.

**Review by lol768 — APPROVED** [review]

**Pull request merged**